CMakeLists.txt

```diff
@@ -394,6 +394,8 @@ SET(ENABLE_NLS 1 CACHE BOOL "Define whether program translation should be suppor
 IF(ENABLE_NLS)
      ADD_DEFINITIONS(-DENABLE_NLS)
 ENDIF()
+# This option toggle manipulations with files in po directory
+SET(ENABLE_PODIR 1 CACHE BOOL "Define whether the manipulation for files of translations should be supported.")
```

Comment:
The "manipulation for files of translations" sounds weird. This should be "processing of translation files" IMO. Or even better, describe what exactly processing is meant (`.po` → `.qm` conversion, what else?)

Reply:
Hm... yeah, the word "processing" is better here.